libs/content-search-toolkit/src/services/elastic.service.ts

```diff
@@ -109,19 +109,28 @@ export class ElasticService {
    * @return {boolean} True iff a document was removed
    * Filter the HUMONGOUS documents in an error object
    */
-  private filterDoc<T>(o: T): boolean {
-    let deleted = false
-    if (Object.keys(o).length == 0) return false
-    for (const key in o) {
-      const value = o[key]
-      // Only HUMONGOUS documents should reach this limit
-      if (typeof value == 'string' && value.length > 10000) {
-        delete o[key]
-        deleted = true
+  private filterDoc<T>(obj: T): boolean {
+    const visited = new Set()
+    const filterDocRecursive = <O>(o: O): boolean => {
```

Comment:
For better readability, is it possible to change the name `o` from a single letter?
Maybe something like `node`, or `leaf`?

Reply:
:sweat_smile: Sorry that was the mathematician in me. Should definitely be more legible than that.

Reply:
I renamed it to node

Reply:
Nice :+1: